test/math/kd_tree_test.cc

```diff
@@ -50,40 +51,38 @@ TEST_F(SquaredDistanceTest, Test) {
   EXPECT_DOUBLE_EQ(squared_distance, (details::SquaredDistance<Vector3, 3>()(v1, v2)));
 }
 
+// Tests maliput::math::details::Node class.
+class NodeTest : public ::testing::Test {};
+
+TEST_F(NodeTest, Test) {
+  const Vector3 left_coord{1., 2., 3.};
+  const Vector3 right_coord{4., 5., 6.};
+  const Vector3 parent_coord{7., 8., 9.};
+  details::Node<Vector3, AxisAlignedBox> left{left_coord};
+  details::Node<Vector3, AxisAlignedBox> right{right_coord};
+  details::Node<Vector3, AxisAlignedBox> parent{parent_coord};
+  EXPECT_EQ(left.get_coordinate(), left_coord);
+  parent.set_index(14.);
+  EXPECT_EQ(parent.get_index(), 14.);
+  left.set_parent(&parent);
+  EXPECT_EQ(left.get_parent(), &parent);
+  parent.set_left(&left);
+  EXPECT_EQ(parent.get_left(), &left);
+  parent.set_right(&right);
+  EXPECT_EQ(parent.get_right(), &right);
+  const AxisAlignedBox box{Vector3{1., 2., 3.}, Vector3{10., 11., 11.}};
+  parent.set_region(std::make_unique<AxisAlignedBox>(Vector3{1., 2., 3.}, Vector3{10., 11., 12.}));
+  EXPECT_EQ(parent.get_region().max_corner(), Vector3(10., 11., 12.));
+  EXPECT_EQ(parent.get_region().min_corner(), Vector3(1., 2., 3.));
```

Comment:
Can you re-order this test so you declare all your constants, initialize the variables to evaluate and then perform the assertions?

Reply:
Done.